tests/scripts/run_tt_eager.py

```diff
@@ -29,14 +29,6 @@
 )
 
 TT_EAGER_COMMON_TEST_ENTRIES = (
-    void_for_gs(TestEntry("tt_eager/tests/ops/ccl/test_all_gather_utils", "ops/ccl/test_all_gather_utils")),
-    void_for_gs(
-        TestEntry(
-            "tt_eager/tests/ops/ccl/test_all_gather_sharded_indexing_helpers",
```

Comment:
Hey @Aswinmcw, where did these go?

Reply:
Hi @SeanNijjar , i have reverted these gtest changes and only moved the pytest to TTNN unit test and modified the file paths in those. Kindly review it once